src/main/java/seedu/address/logic/commands/CompleteCommand.java

```diff
@@ -161,7 +161,7 @@ private String completeOneTaskReturnStringOfTask(Index targetIndex,
             throw new CommandException(MESSAGE_ALREADY_COMPLETED);
         }
 
-        modelToUpdate.updateTask(taskToComplete, completedTask);
+        modelToUpdate.updateTaskStatus(taskToComplete, completedTask);
         return completedTask.toString();
```

Comment:
There are more instances in this file where the usage of updateTask has not been changed to updateTaskStatus

Reply:
I couldn't find more cases though

Reply:
Was referring to line 135, i think you changed it in your new commit so all's good